test/runtests.jl

```diff
@@ -1,24 +1,21 @@
 using Dispatcher
 using ResultTypes
-using Base.Test
+using Compat.Test
 using Memento
 using IterTools
-
+using Compat.Distributed
+using DeferredFutures
 import LightGraphs
 
 using Compat: @__MODULE__
 using ResultTypes: iserror
 
-const LOG_LEVEL = "info"      # could also be "debug", "notice", "warn", etc
-
-Memento.config(LOG_LEVEL; fmt="[{level} | {name}]: {msg}")
 const logger = getlogger(@__MODULE__)
 
 function test_addproc(x::Int; level=LOG_LEVEL)
     ret = addproc(x)
-    @everywhere using Dispatcher
-    @everywhere using Memento
-    @everywhere Memento.config(level; fmt="[{level} | {name}]: {msg}")
```

Comment:
Just to clarify, we're dropping this because Memento defaults to info level?

Reply:
I believe that was the argument, yes

Reply:
I think we shouldn't actually drop this, we should change it to be `Memento.config(level)`.

Reply:
And also set the default kwarg to "info", because right now it's an UndefVarError